lib/discussion.ts

```diff
@@ -10,38 +13,121 @@ const GH_DATA = join(
   process.env.DATA_REPO ?? "data-repo",
   "../data/github/discussions",
 );
-export const categoriesMap = new Map();
+export const categoriesArray: { name: string; emoji: string }[] = [];
```

Comment:
Just keeping it `categories` would be ideal. We know implicitly it's an array by reading it.